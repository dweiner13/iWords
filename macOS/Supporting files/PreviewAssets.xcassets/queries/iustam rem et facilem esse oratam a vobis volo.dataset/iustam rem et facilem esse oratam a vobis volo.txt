iust.am              ADJ    1 1 ACC S F POS             
justus, justa -um, justior -or -us, justissimus -a -um  ADJ  
just, fair, equitable; right, lawful, justified; regular, proper;

r.em                 N      5 1 ACC S F                 
res, rei  N (5th) F  
thing; event/affair/business; fact; cause; property; [~ familiaris => property]

et                   CONJ                               
et  CONJ  
and, and even; also, even;  (et ... et = both ... and);

facil.em             ADJ    3 2 ACC S C POS             
facilis, facile, facilior -or -us, facillimus -a -um  ADJ  
easy, easy to do, without difficulty, ready, quick, good natured, courteous;

ess.e                VPAR   3 1 VOC S M PERF PASSIVE PPL
edo, edere, edi, essus  V (3rd) TRANS  
es.se                V      7 3 PRES ACTIVE  INF 0 X      Early   
edo, esse, -, -  V TRANS  
eat/consume/devour; eat away (fire/water/disease); destroy; spend money on food
.esse                V      5 1 PRES ACTIVE  INF 0 X    
sum, esse, fui, futurus  V  
be; exist; (also used to form verb perfect passive tenses) with NOM PERF PPL

orat.am              VPAR   1 1 ACC S F PERF PASSIVE PPL
oro, orare, oravi, oratus  V (1st)  
beg, ask for, pray; beseech, plead, entreat; worship, adore;

a                    N      9 8 X   X M                 
A., abb.  N M  
Aulus (Roman praenomen); (abb. A./Au.); [Absolvo, Antiquo => free, reject];
a., abb.  N M    lesser
year; abb. ann./a.; [regnavit a(nnis). XLIIII => he reigned for 44 years];
a                    PREP   ABL                         
a  PREP  ABL  
by (agent), from (departure, cause, remote origin/time); after (reference);
a                    PREP   ACC                         
a  PREP  ACC  
ante, abb. a.; [in calendar expression a. d. = ante diem => before the day];
a                    INTERJ                             
a  INTERJ  
Ah!; (distress/regret/pity, appeal/entreaty, surprise/joy, objection/contempt);

v.obis               PRON   5 3 DAT P C                 
v.obis               PRON   5 3 ABL P C                 
you (pl.), ye;

vol.o                V      6 2 PRES ACTIVE  IND 1 S    
volo, velle, volui, -  V  
wish, want, prefer; be willing, will;
vol.o                V      1 1 PRES ACTIVE  IND 1 S    
volo, volare, volavi, volatus  V (1st)  
fly;
volo                 N      3 1 NOM S M                 
volo                 N      3 1 VOC S M                 
volo, volonis  N (3rd) M    uncommon
volunteers (pl.); (in the Second Punic War);

