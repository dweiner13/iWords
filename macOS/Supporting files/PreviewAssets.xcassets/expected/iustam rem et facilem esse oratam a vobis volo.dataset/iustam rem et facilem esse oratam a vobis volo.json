[{"definition":{"_0":{"words":[{"meaning":"just, fair, equitable; right, lawful, justified; regular, proper;","expansion":{"adj":{"_0":"justus, justa -um, justior -or -us, justissimus -a -um","_1":[]}}}],"truncated":false,"possibilities":[{"adjective":{"_0":{"number":"S","gender":"F","variety":1,"degree":"POS","text":"iust.am","declension":1,"case":"ACC"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"thing; event\/affair\/business; fact; cause; property; [~ familiaris => property]","expansion":{"noun":{"_1":5,"_0":"res, rei","_3":[],"_2":"F"}}}],"truncated":false,"possibilities":[{"noun":{"_0":{"declension":5,"case":"ACC","number":"S","gender":"F","variety":1,"text":"r.em"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"and, and even; also, even;  (et ... et = both ... and);","expansion":{"conj":{"_0":"et","_1":[]}}}],"truncated":false,"possibilities":[{"conjunction":{"_0":"et"}}]}}},{"definition":{"_0":{"words":[{"meaning":"easy, easy to do, without difficulty, ready, quick, good natured, courteous;","expansion":{"adj":{"_0":"facilis, facile, facilior -or -us, facillimus -a -um","_1":[]}}}],"truncated":false,"possibilities":[{"adjective":{"_0":{"number":"S","gender":"C","variety":2,"degree":"POS","text":"facil.em","declension":3,"case":"ACC"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"eat\/consume\/devour; eat away (fire\/water\/disease); destroy; spend money on food","expansion":{"verb":{"_0":"edo, esse, -, -","_2":["trans."]}}}],"truncated":false,"possibilities":[{"verbParticiple":{"_0":{"gender":"M","number":"S","variety":1,"conjugation":3,"tense":"PERF","text":"ess.e","voice":"PASSIVE","case":"VOC"}}},{"verb":{"_0":{"tense":"PRES","mood":"INF","voice":"ACTIVE","variety":3,"text":"es.se"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"be; exist; (also used to form verb perfect passive tenses) with NOM PERF PPL","expansion":{"verb":{"_0":"sum, esse, fui, futurus","_2":[]}}}],"truncated":false,"possibilities":[{"verb":{"_0":{"tense":"PRES","mood":"INF","voice":"ACTIVE","variety":1,"text":".esse"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"beg, ask for, pray; beseech, plead, entreat; worship, adore;","expansion":{"verb":{"_0":"oro, orare, oravi, oratus","_2":[],"_1":1}}}],"truncated":false,"possibilities":[{"verbParticiple":{"_0":{"gender":"F","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"orat.am","voice":"PASSIVE","case":"ACC"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"Aulus (Roman praenomen); (abb. A.\/Au.); [Absolvo, Antiquo => free, reject];","expansion":{"noun":{"_0":"A., abb.","_2":"M","_3":[]}}},{"meaning":"year; abb. ann.\/a.; [regnavit a(nnis). XLIIII => he reigned for 44 years];","expansion":{"noun":{"_0":"a., abb.","_2":"M","_3":["lesser"]}}}],"truncated":false,"possibilities":[{"noun":{"_0":{"variety":8,"text":"a","gender":"M"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"by (agent), from (departure, cause, remote origin\/time); after (reference);","expansion":{"prep":{"_0":"a","_2":[],"_1":"ABL"}}}],"truncated":false,"possibilities":[{"preposition":{"_0":{"text":"a","case":"ABL"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"ante, abb. a.; [in calendar expression a. d. = ante diem => before the day];\na                    INTERJ                             \na  INTERJ  \nAh!; (distress\/regret\/pity, appeal\/entreaty, surprise\/joy, objection\/contempt);","expansion":{"prep":{"_0":"a","_2":[],"_1":"ACC"}}}],"truncated":false,"possibilities":[{"preposition":{"_0":{"text":"a","case":"ACC"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"you (pl.), ye;"}],"truncated":false,"possibilities":[{"pronoun":{"_0":{"declension":5,"case":"DAT","number":"P","gender":"C","variety":3,"text":"v.obis"}}},{"pronoun":{"_0":{"declension":5,"case":"ABL","number":"P","gender":"C","variety":3,"text":"v.obis"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"wish, want, prefer; be willing, will;","expansion":{"verb":{"_0":"volo, velle, volui, -","_2":[]}}}],"truncated":false,"possibilities":[{"verb":{"_0":{"person":1,"number":"S","tense":"PRES","variety":2,"text":"vol.o","voice":"ACTIVE","mood":"IND"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"fly;","expansion":{"verb":{"_0":"volo, volare, volavi, volatus","_2":[],"_1":1}}}],"truncated":false,"possibilities":[{"verb":{"_0":{"person":1,"number":"S","tense":"PRES","conjugation":1,"variety":1,"text":"vol.o","voice":"ACTIVE","mood":"IND"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"volunteers (pl.); (in the Second Punic War);","expansion":{"noun":{"_1":3,"_0":"volo, volonis","_3":["uncommon"],"_2":"M"}}}],"truncated":false,"possibilities":[{"noun":{"_0":{"declension":3,"case":"NOM","number":"S","gender":"M","variety":1,"text":"volo"}}},{"noun":{"_0":{"declension":3,"case":"VOC","number":"S","gender":"M","variety":1,"text":"volo"}}}]}}}]